.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish latest build
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  deploy:
+    if: ${{ github.repository == 'SpoonLabs/gumtree-spoon-ast-diff' }}
+    name: 'Publish package'
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Wait for tests to pass
+      uses: lewagon/wait-on-check-action@v1.1.1
+      with:
+        ref: ${{ github.ref }}
+        running-workflow-name: 'Publish package'
```

Comment:
'Publish package' is not a valid name in ci.yml. Is that a problem?

Reply:
The `wait for tests to pass` waits for the `ci.yml` to pass and only then `publish.yml` will execute on master. "Publish package" is the name of the current job so specifying that will ensure that all previous workflows are completed before publish.yml is initiated. Reference: https://github.com/lewagon/wait-on-check-action#running-workflow-name